frontend/simple/components/ProgressOverview.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <section>
+    <div class="level">
+      <h3 class="title"><i18n>{{ period }} Overview</i18n></h3>
+    </div>
+      <div class="level">
+        <dl class="level-left summary" :style="{ color: themeColor }">
```

Comment:
I guess this will also rather be handled by a theme class rather than a style attribute as in #373 right? or should theming be 100% flexible?

Reply:
You are right, I've leaved it as it is to better  rethink it on a next iteration.